### PR TITLE
Create candidate-asthinasthi.md

### PR DESCRIPTION
**Pull request by asthinasthi**
# Changes
Application to the steering committee

### CODE REVIEW

File: elections/2022-SC-EU/candidate-asthinasthi.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  - affiliation: Adobe
+-------------------------------------------------------------
+
+Here's a paragraph about my contributions to Knative.
+I have no contributions but am an end user.
+
+Here's a paragraph about why I'm running:
+
+At Adobe I worked on Internal Developer Platforms. We are a Kubernetes shop. I work on CICD pipelines for serverless applications. Recently we have adopted 
+ArgoCD for serverless CICD pipelines. We plan to roll out KNative to our end users and migrate them from AWS and Azure Functions.
```
Comment (zroubalik) [inline elections/2022-SC-EU/candidate-asthinasthi.md:14]:
```suggestion
ArgoCD for serverless CICD pipelines. We plan to roll out Knative to our end users and migrate them from AWS and Azure Functions.
```
Just a nit :)

**Comment by csantanapr** [comment]
@asthinasthi thank you for nominating your self for Steering End User seat.
You submitted the nomination after the deadline for nominations stated in our [doc](https://github.com/knative/community/tree/main/elections/2022-SC-EU#schedule) dead line was Jan 26th, you submitted Jan 27th.
With this said I speaking for all steering (@lance @itsmurugappan @salaboy @nainaz) that we would accept your late nomination if you could fix in the next 24th hour the rejection based on missing CNCF CLA. 
Steering will be deliberating who is appointed this week, if this PR gets merged we could include you in the pool.

**Comment by csantanapr** [comment]
One more thing @asthinasthi Adobe is not listed in the [Knative ADOPTERS.MD](https://github.com/knative/community/blob/main/ADOPTERS.MD) file, this is requirement to be able to be candidate to end user seat.
Again in the next 24 hours you could address this two items we will be glad to consider your as a candidate
1. Nomination (missing CNCF CLA)
2. ADOPTERS.MD

File: elections/2022-SC-EU/candidate-asthinasthi.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+-------------------------------------------------------------
+name: Your Name
```
Comment (csantanapr) [inline elections/2022-SC-EU/candidate-asthinasthi.md:2]: You need to provide your full name

**Comment by csantanapr** [comment]
closing as candidate's company is not the [ADOPTERS.MD](https://github.com/knative/community/blob/main/ADOPTERS.MD) file as required in the [nomination instructions](https://github.com/knative/community/blob/main/elections/2022-SC-EU/README.md)

/close

**Comment by asthinasthi** [comment]
Thank you, 
How does one get into Adopters List?
I did add a comment? https://github.com/knative/community/issues/696#issuecomment-1406116974 

Do I need an approval from my company?

**Comment by dprotaso** [comment]
Hey @asthinasthi ADOPTERS list is just a file in this repository - you can make a PR like so https://github.com/knative/community/pull/1338

**Pull request closed without merging**